pom.xml

```diff
@@ -35,14 +35,14 @@
     <jdk.version>1.8</jdk.version>
     <joda-time.version>2.10.1</joda-time.version>
     <json-simple.version>1.1.1</json-simple.version>
-    <karaf.version>4.2.0</karaf.version>
+    <karaf.version>4.2.2</karaf.version>
     <mina.version>2.0.19</mina.version>
     <node.version>v8.12.0</node.version>
     <npm.version>6.4.1</npm.version>
     <osgi.compendium.version>5.0.0</osgi.compendium.version>
     <osgi.core.version>5.0.0</osgi.core.version>
     <osgi.enterprise.version>5.0.0</osgi.enterprise.version>
-    <pax.web.version>7.0.0</pax.web.version>
+    <pax.web.version>7.2.3</pax.web.version>
```

Comment:
The news article (and pom) for the Karaf 4.2.2 release list pax web v7.2.5 (though confusingly the release note mention v7.2.4).

pax web 7.2.3 was supported by karaf 4.2.1

Reply:
Not sure where I did take the version from. The notes about the versions are always kind of messy :-(
I'll look into it and fix it.

Reply:
Do we need to specify it at all if Karaf is pulling it in already?

Reply:
> Do we need to specify it at all if Karaf is pulling it in already?

Good question and it seems like we don't need it. The specified version was used at a few more places but cleaning that out results in a fully functional system so I updated this pull request accordingly.